### DISCUSSION TITLE
How to run WDI5 spec files from different WEBAPP's in SAP CAP repository

### DISCUSSION THREAD

**Discussion by smrtyviks** [ui5-community/wdi5#539] [Q&A]
Hi All,

I am trying to setup WDI5 framework to add some Test Script to our SAP CAP front end repository which has multiple webapps and each webapp is a project in itself, So for normally executing my Test scripts, I first start the server and than try to run my scripts. 
And i have to be in the Porject directory to perform above steps.

Does anyone have a better idea on how i could execute the Test Suite which will run all the tests inside multiple project directories?

Regards,
Vikrant

**Comment by vobu** [comment]
hi, unfortunately there' way too few context in your question to recommend the best way 🔮 . Plus it seems this is more a question on your overall test setup than a question specifically related to `wdi5`. I'd recommend that you make use of [the commercial support available for `wdi5`](https://github.com/ui5-community/wdi5/blob/main/SUPPORT.md) → that way you get direct interaction with experts helping you improve and resolve things.
